corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -741,15 +742,15 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
                 }
             });
             if (invalidRequiredFields.length === 0) {
-                self.performSubmit("dynamicSearch");
```

Comment:
I see you updated the line that references this, but just checking whether there's anything else that we should consider when changing this string?  Given the context of this change, I'm guessing it's not used in metrics that might break?

Reply:
This was introduced as part of this [PR](https://github.com/dimagi/commcare-hq/pull/33678). No it's not used directly in metrics. It's just used [here](https://github.com/dimagi/commcare-hq/blob/f75e09964a61d8594f3fccf5a787b51eb97be46e/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js#L175) to record the metric if relevant.